yaml.lint.cloudbuild.yaml

```diff
@@ -4,9 +4,17 @@ steps:
   entrypoint: '/bin/sh'
   args:
   - -c
-  - | # Recursively find yaml files (ignored files in .yamllint config)
-    find ./* -name '*.yaml' -not -path "./testing/*" \
-      -exec yamllint -f parsable {} \; \
-      | tee /proc/1/fd/1 \
-      | grep -q 'error'
-    if [[ $? -eq 0 ]] ; then exit 1 ; fi
+  - | # Recursively find yaml files (yamllint must be run in directory)
+    find . -name "*.yaml" -not -path "./testing/*" -not -path "*.lint.cloudbuild.yaml" \
+      -exec sh -c '
+      root=$(pwd)
+      lint () {
```

Comment:
I think defining the function to be called before the find would make the whole thing much more readable.

Reply:
It seems like there are limitations around moving the function outside the find when using shell (https://unix.stackexchange.com/questions/50692/executing-user-defined-function-in-a-find-exec-call)